Library/Formula/libmill.rb

```diff
@@ -0,0 +1,45 @@
+class Libmill < Formula
+  desc "Go-style concurrency in C"
+  homepage "http://libmill.org/"
+  url "http://libmill.org/libmill-0.16-beta.tar.gz"
+  sha256 "c6d68d8291dadf2ee111f6ceed057f2fd267e7dc334bef8d15b6316c35b6a0ca"
+  head "https://github.com/sustrik/libmill.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "cmake" => :build
+
+  def install
+    ENV.O1
```

Comment:
The issue was known before I made this recipe. That was why I used env :std and switched to ENV.O1 (for a superenv).

See https://github.com/sustrik/libmill/issues/21

A comment has been added to the recipe. The fault is a Clang bug where it reuses a stack allocated address (which shouldn't happen and does not happen in other compilers)